main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/input"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+// Config ...
+type Config struct {
+	IpaPath           string          `env:"ipa_path"`
+	PkgPath           string          `env:"pkg_path"`
+	ItunesConnectUser stepconf.Secret `env:"itunescon_user,required"`
+	Password          stepconf.Secret `env:"password"`
+	AppPassword       stepconf.Secret `env:"app_password"`
+}
+
+func main() {
+	var cfg Config
+	if err := stepconf.Parse(&cfg); err != nil {
+		log.Errorf("Error: %s\n", err)
+		os.Exit(1)
+	}
+
+	stepconf.Print(cfg)
+	fmt.Println()
+
+	if err := cfg.validateEnvs(); err != nil {
+		failf("input error: %s", err)
```

Comment:
final error message should start with capital case (if you are logging with capital case in the step)